source/_integrations/nibe_heatpump.markdown

```diff
@@ -0,0 +1,80 @@
+---
+title: Nibe Heat Pump
+description: Instructions on how to integrate a Nibe Heat Pump into Home Assistant.
+ha_category:
+  - Climate
+ha_release: 2022.10
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@elupus'
+ha_domain: nibe_heatpump
+ha_ssdp: false
+ha_platforms:
+  - sensor
+  - binary_sensor
+  - number
+  - switch
+  - select
+  - climate
+  - water_heater
```

Comment:
These two are yet to be reviewed/merged. I will remove these from this pull should they not make the release.

Reply:
```suggestion
```